platform-emf/src/main/java/com/softicar/platform/emf/form/refresh/EmfFormInteractiveRefreshSpan.java

```diff
@@ -63,8 +67,8 @@ public RefreshButton() {
 
 		private void queueEntityForRefreshAndCancelEditMode() {
 
-			formBody.queueEntityForRefresh();
-			formBody.cancelEditMode();
```

Comment:
canceling the edit mode here was causing repeated displays of the interactive refresh element to be shown in some cases, so I moved the whole logic into that new method `refreshAfterConcurrentModification`